dependencies/dl/meson.build

```
@@ -0,0 +1,9 @@
+# libdl Dependency File
+#============================================================================
+
+
+# Look for libdl, a required dependency.
+#--------------------------------------------------
+libdl_dep = declare_dependency(
+    dependencies : cpp.find_library('dl'),
+    link_args :  get_option('dl_link_args'))
```

```suggestion
    link_args :  get_option('dl_link_args'))
    
```
Lacks the extra EOF at EOL.

---

P.S.: do this project uses some form of [Editorconfig](https://editorconfig.org/) file?

---

editorconfig file is here

https://github.com/NixOS/nix/blob/master/.editorconfig